features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -4071,9 +4071,29 @@
                     <Path>/oauth2/introspect</Path>
                     <Path>/oauth2/revoke</Path>
                     <Path>/oauth2/userinfo</Path>
+                    <Path>/oauth2/authorize</Path>
+                    <Path>/oauth2/jwks</Path>
                 </SubPaths>
             </Context>
+            <Context>
+                <BasePath>/authenticationendpoint/</BasePath>
+                <BasePath>/accountrecoveryendpoint/</BasePath>
+                <BasePath>/accounts/</BasePath>
```

Comment:
Shall we update seperate contexts in xml.j2 as well

Reply:
Updated